dispatcherd/service/process.py

```diff
@@ -99,10 +99,17 @@ async def read_finished(self) -> dict[str, Union[str, int]]:
         message = await self.get_event_loop().run_in_executor(None, self.finished_queue.get)
         return message
 
+    def shutdown(self) -> None:
+        self.finished_queue.close()
```

Comment:
Tests are working but I'm seeing an order-of-operations error.

```
ERROR:dispatcherd.service.main:Pool manager encountered error
Traceback (most recent call last):
  File "/home/alancoding/repos/dispatcher/dispatcherd/service/main.py", line 103, in shutdown
    await self.pool.shutdown()
  File "/home/alancoding/repos/dispatcher/dispatcherd/service/pool.py", line 454, in shutdown
    self.process_manager.finished_queue.put('stop')
  File "/usr/lib64/python3.12/multiprocessing/queues.py", line 88, in put
    raise ValueError(f"Queue {self!r} is closed")
ValueError: Queue <multiprocessing.queues.Queue object at 0x7f4cd2fa7e30> is closed
```

Shouldn't be too hard to figure out.